ChatApp/View/RecipientCapsuleBubble.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  BubbleView.swift
+//  chatApp.test
+//
+//  Created by Ana Dzebniauri on 20.04.23.
+//
+
+import UIKit
+
+// MARK: - Bubbles
+
+class RecipientCapsuleBubble: UIView {
+    
+    private var radius: CGFloat {
+        bounds.height/2
```

Comment:
missing spacing: "bounds.height / 2"
+ as others mentioned in the meeting radius should be equal to the size that is given in the design. while using height / 2 your View's radius will change based on its size which is not correct.